Sources/Client/Views/Config/SettingsView.swift

```diff
@@ -1,27 +1,58 @@
 import SwiftUI
 import DeltaCore
 
+enum SettingsState {
+  case none, accounts, plugins, update, troubleshooting, video
+}
+
 struct SettingsView: View {
   var isInGame: Bool
   var eventBus: EventBus?
   var done: () -> Void
   
-  init(isInGame: Bool, eventBus: EventBus?, onDone done: @escaping () -> Void) {
+  /// The `NavigationLink` to be selected on initializaiton
+  @State private var initialLandingPage: SettingsState? = nil
+  
+  init(isInGame: Bool,
```

Comment:
Please style multi line function signatures like I do elsewhere in the code, see https://github.com/stackotter/delta-client/blob/52b93a0844faeae76ea11153d41817c0bf54c2b9/Sources/Core/Sources/Resources/Block/Model/Intermediate/IntermediateBlockModelFace.swift#L21

Reply:
yep sorry, pressed on "commit suggestions" on one of the proposals you made before but indentation apparently got messed up, will fix it by hand